bang/templates/items/costumeItem.html

```diff
@@ -1,10 +1,24 @@
 {% with costume=item %}
+{% if view == 'chibis' and costume.cached_chibis %}
+<a style="display: block;"
+    {% if costume.card %}
+    href="{{ costume.card.item_url }}" data-ajax-url="{{ costume.card.ajax_item_url }}" data-ajax-title="{{ costume }}" class="panel panel-{{ costume.card.english_attribute }} text-center"
+    {% else %}
+    href="{{ costume.item_url }}" data-ajax-url="{{ costume.ajax_item_url }}" data-ajax-title="{{ costume }}" class="panel panel-secondary text-center"
+    {% endif %}
+>
+    {# above: link to cards instead of live2d if we can, to be like the old chibi view #}
+    <div class="panel-heading">{{ costume }}</div>
+    <div class="panel-content padding20"><div>{% for chibi in costume.cached_chibis %}<img src="{{ chibi.image_url }}" alt="chibi" width="33%" />{% endfor %}</div></div>
```

Comment:
Tbh I'm not a big fan of that chibi view I made on a whim, so if you want to make it look different, that's totally fine!

Reply:
it looks ok except for the chibis' sizes being different. i can't think of any way to fix that without reuploading them all, but i might revisit it later.